docs/documentation/gui.md

```diff
@@ -18,30 +18,30 @@ To install the Dimlpfidex GUI, follow these steps:
 !!! tip "Installation guide for..."
 
     === "Windows"
-        1. **Download** the archive by opening your any browser and [downloading the latest release](https://github.com/HES-XPLAIN/dimlpfidex/releases/)
-        2. **Decompress** the achive with any file archiver, place the folder where you can find it afterwards (Desktop, Documents, etc...).
+        1. **Download** the archive by opening any browser and [downloading the latest release](https://github.com/HES-XPLAIN/dimlpfidex/releases/)
+        2. **Decompress** the archive with any file archiver inside any folder where you can find afterward (`Desktop/`, `Documents/`, etc...).
         3. **Execute** the program by double-clicking on the `dimlpfidex_gui.exe` file.
-        4. Done !
+        4. Done!
 
     === "Linux"
-        1. **Download** the archive by opening your any browser and [downloading the latest release](https://github.com/HES-XPLAIN/dimlpfidex/releases/)
-        2. **Decompress** the achive with any file archiver and place the folder where you can find it afterwards (Desktop, Documents, etc...).
+        1. **Download** the archive by opening any browser and [downloading the latest release](https://github.com/HES-XPLAIN/dimlpfidex/releases/)
+        2. **Decompress** the archive with any file archiver inside any folder where you can find afterward (`Desktop/`, `Documents/`, etc...).
```

Comment:
Same here, "Decomplress teh archvive" The target audience of the doc should know about basic stuff already.